src/config/configure.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types';
+
+let service = null;
+
+export default function configure(ConfigService, services) {
+  service = new ConfigService(services);
+  PropTypes.checkPropTypes({
```

Comment:
Is this working?

Reply:
Wow.  Uh.  Sure do.  I failed to finish that code. ;)  Good catch.

Reply:
Oh interesting you're using this to validate the implementation. Neat.